assets/dev/js/editor/document/save/commands/internal/set-is-modified.js

```diff
@@ -0,0 +1,23 @@
+import CommandInternalBase from 'elementor-api/modules/command-internal-base';
+
+export class SetIsModified extends CommandInternalBase {
+	validateArgs( args ) {
+		this.requireArgumentType( 'status', 'boolean', args );
+	}
+
+	apply( args ) {
+		const { status } = args,
```

Comment:
Keep the standard the `document` can be an `arg`
```suggestion
		const { status, document = elementor.documents.getCurrent() } = args,
```